docker/yarn/clean_environment.bash

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+
+image_name="cluster/hadoop"
+image_name="$image_name:new"
+
+network_name="hadoop"
+
+echo -e "Deleting older containers and network\n"
+
+# Stop all containers related to the image
+docker ps -a | awk '{ print $1,$2 }' | grep $image_name | awk '{print $1 }' | xargs -I {} docker stop {}
+
+# Remove all containers related to the image
+docker ps -a | awk '{ print $1,$2 }' | grep $image_name | awk '{print $1 }' | xargs -I {} docker rm {}
+
+# Remove the relevant image to prevent dangling
+# docker rmi $image_name
```

Comment:
why is this commented out? Should you remove the docker image as well?

Reply:
We retain the image so that the hadoop-docker cluster can be remade. For eg:
```bash
bash build_docker_image.bash
bash create_cluster.bash 7
bash stop_cluster.bash
# remake cluster
bash create_cluster.bash 4
bash stop_cluster.bash
```
If you specifically think that we don't need to remake a cluster, then that specific line(docker rmi $image_name) can be uncommented